firebase-dataconnect/src/main/kotlin/com/google/firebase/dataconnect/serializers/TimestampSerializer.kt

```diff
@@ -71,9 +71,7 @@ public object TimestampSerializer : KSerializer<Timestamp> {
     val regex =
       Regex("^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}(\\.\\d{0,9})?(Z|[+-]\\d{2}:\\d{2})$")
 
-    require(strUppercase.matches(regex)) {
-      "Value does not conform to the RFC3339 specification (str=$str)"
-    }
+    require(strUppercase.matches(regex)) { "Timestamp string: (str=$str) format doesn't support." }
```

Comment:
This updated error message is less actionable than the old one, as it doesn't give any hints as to what the problem was. If the only digression from RFC3339 is arbitrary sub-second support, then maybe we could just change the old message to something like this:

> Value does not conform to the RFC3339 specification with up to 9 digits of sub-second precision (str=$str)

What do you think?

See go/error-message-best-practices